### PR TITLE
fix(CreditCardExpirationField): fix wrong formatting when editing field's value

### PR DESCRIPTION
**Pull request by marcoskolodny**
Issue: [Link](https://jira.tid.es/browse/WEB-1631)

### CODE REVIEW

File: src/__acceptance_tests__/input-fields-acceptance-test.tsx [hunk 1]
```diff
@@ -117,6 +117,16 @@ test.each(STORY_TYPES)('CreditCardExpirationField (%s)', async (storyType) => {
...
+
+    await clearAndType(page, field, '222');
+    expect(await getValue(field)).toBe('02/22');
+
+    await page.keyboard.press('Backspace');
+    await page.keyboard.press('Backspace');
+    expect(await getValue(field)).toBe('02');
+
+    await field.type('22');
+    expect(await getValue(field)).toBe('02/22');
```
Comment (atabel) [inline src/__acceptance_tests__/input-fields-acceptance-test.tsx:129]: is this the case we try to fix?

trying this in master, the result is almos the same

**Review by yceballost — APPROVED** [review]

**Review by atabel — APPROVED** [review]

**Comment by pladaria** [comment]
Consider the proposed changes from our teams conversation

**Comment by marcoskolodny** [comment]
> Consider the proposed changes from our teams conversation

@pladaria updated the logic :)

**Review by atabel — APPROVED** [review]

**Review by pladaria — APPROVED** [review]
thanks!

**Pull request merged**